### ISSUE TITLE
Consider record compression for repeated characters

### ISSUE THREAD

**Issue by trevor-vaughan** [Scribery/tlog#197]
I noticed that my logs are getting a LOT of data that consists of just the `\` character repeated.

I would suggest making that a multiplier field so that there isn't so much data hitting the logs constantly.

Something like `%tbuf(100*)\` or the like would a lot of space, particularly when using tlog with multiple hosts.

**Comment by spbnick** [comment]
@trevor-vaughan lots of `\` characters repeated might be you recording yourself viewing the logs ;) Happened to me more than once. That said, optional compression might be good, as long as the logs get uncompressed for searching somewhere (e.g. in Elasticsearch).

**Comment by trevor-vaughan** [comment]
@spbnick Yup, that was it. It's going to be really common though since one of the most commonly audited accounts is `root`. I'd still like the feature :-D
